technologies/backend/firebase1.md

```diff
@@ -15,8 +15,9 @@ A developer who is able to use Firebase to develop data-driven web and mobile ap
 ## Suggested Learning:
 
 * [Firebase Docs and Tutorial](https://firebase.google.com/docs/web/setup)
-* [Firebase in a Weekend](https://www.udacity.com/course/firebase-in-a-weekend-by-google-android--ud0352)
-* [Introducing Firebase](https://www.youtube.com/playlist?list=PLl-K7zZEsYLmOF_07IayrTntevxtbUxDL)
+* [Firebase FireStore tutorial= (https://www.youtube.com/watch?v=4d-gIPGzmK4&list=PL4cUxeGkcC9itfjle0ji1xOZ2cjRGY_WB)
+* [Firebase Auth tutorial] = (https://www.youtube.com/watch?v=aN1LnNq4z54&list=PL4cUxeGkcC9jUPIes_B8vRjn1_GaplOPQ).
+
```

Comment:
The markdown format in both your links is wrong. The name of the link should be inside brackets like [this]. The link url should be in parenthesis like (this). There should be no spaces or characters between the name and link. Take a look at the link above that is formatted correctly.

Reply:
Great job!